osCam/core/models.py

```diff
@@ -1,13 +1,9 @@
 #core/models.py
-
 from django.db import models
-from django.contrib.auth.models import User
```

Comment:
were these imports not being used?

Reply:
no they were not being used. so I took them out